Automattic/wp-calypso
# Add webauthn support for the two step re-auth dialog

### Changes proposed in this Pull Request

* Adds support for security keys on the `2fa-reauth-dialog` block. 

### Testing instructions

* Add a security key to your WP.com account.
* Navigate to `/me` and remove the `twostep_auth` cookie from the browser console.
* Reload the page, and notice the `2fa-reauth-dialog` is shown along with a new option to authenticate using a hardware key. 

### Screenshots
#### Re-auth dialog
**Before**
<img  width="436" src="https://user-images.githubusercontent.com/1081870/74947345-b399ff00-53c8-11ea-99b7-a47073a4d6f9.png" />

**After**
<img  width="436" src="https://user-images.githubusercontent.com/1081870/74947405-cf050a00-53c8-11ea-8ed2-741c6a1cc437.png" />

when someone clicks the `Continue with security key` button

<img  width="436" src="https://user-images.githubusercontent.com/1081870/74947382-c14f8480-53c8-11ea-9e8d-a64f3df8fc84.png" />
